android-client-sdk/src/main/java/com/devcycle/sdk/android/model/PopulatedUser.kt

```diff
@@ -2,54 +2,64 @@
 import io.swagger.v3.oas.annotations.media.Schema
 import java.util.*
 import android.content.pm.PackageInfo
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.google.gson.annotations.SerializedName
 import kotlin.IllegalArgumentException
 
-@JsonInclude(JsonInclude.Include.NON_NULL)
 internal data class PopulatedUser constructor(
     @Schema(required = true, description = "Unique id to identify the user")
-    @JsonProperty("user_id")
+    @SerializedName("user_id")
     val userId: String,
-    @JsonProperty("isAnonymous")
+    @SerializedName("isAnonymous")
     val isAnonymous: Boolean = false,
     @Schema(description = "User's email used to identify the user on the dashboard / target audiences")
+    @SerializedName("email")
     val email: String? = null,
     @Schema(description = "User's name used to identify the user on the dashboard / target audiences")
+    @SerializedName("name")
     val name: String? = null,
     @Schema(description = "User's language in ISO 639-1 format")
+    @SerializedName("language")
     val language: String? = null,
     @Schema(description = "User's country in ISO 3166 alpha-2 format")
+    @SerializedName("country")
     val country: String? = null,
     @Schema(description = "App Version of the running application")
+    @SerializedName("appVersion")
     val appVersion: String? = null,
     @Schema(description = "App Build number of the running application")
+    @SerializedName("appBuild")
     val appBuild: Long? = null,
     @Schema(description = "User's custom data to target the user with, data will be logged to DevCycle for use in dashboard.")
+    @SerializedName("customData")
     val customData: Map<String, Any>? = null,
     @Schema(description = "User's custom data to target the user with, data will not be logged to DevCycle only used for feature bucketing.")
+    @SerializedName("privateCustomData")
     val privateCustomData: Map<String, Any>? = null,
     @Schema(description = "Date the user was created, Unix epoch timestamp format")
+    @SerializedName("createdDate")
     val createdDate: Long = Calendar.getInstance().time.time,
     @Schema(description = "Platform the Client SDK is running on")
+    @SerializedName("platform")
     val platform: String = "Android",
     @Schema(description = "Version of the platform the Client SDK is running on")
+    @SerializedName("platformVersion")
     val platformVersion: String = Build.VERSION.RELEASE,
     @Schema(description = "User's device model")
+    @SerializedName("deviceModel")
     val deviceModel: String = Build.MODEL,
     @Schema(description = "DevCycle SDK type")
+    @SerializedName("sdkType")
     val sdkType: String = "mobile",
     @Schema(description = "DevCycle SDK Version")
+    @SerializedName("sdkVersion")
     val sdkVersion: String = BuildConfig.VERSION_NAME,
     @Schema(description = "Date the user was last seen, Unix epoch timestamp format")
+    @SerializedName("lastSeenDate")
```

Comment:
does gson not have a way to infer this automatically? seems weird that jackson can do this but gson can't

Reply:
it isn't required, but better to apply it consistently across the board as I found it